fitlins/interfaces/bids.py

```diff
@@ -250,20 +263,21 @@ def _run_interface(self, runtime):
             if len(bold_file) == 0:
                 raise FileNotFoundError(
                     "Could not find BOLD file in {} with entities {}"
-                    "".format(self.inputs.bids_dirs, selectors))
+                    "".format(self.inputs.bids_dir, selectors))
             elif len(bold_file) > 1:
                 raise ValueError(
                     "Non-unique BOLD file in {} with entities {}.\n"
                     "Matches:\n\t{}"
-                    "".format(self.inputs.bids_dirs, selectors,
+                    "".format(self.inputs.bids_dir, selectors,
                               "\n\t".join(
                                   '{} ({})'.format(
                                       f.filename,
                                       layout.files[f.filename].entities)
                                   for f in bold_file)))
 
             # Select exactly matching mask file (may be over-cautious)
-            bold_ents = layout.parse_file_entities(bold_file[0].filename)
+            bold_ents = layout.parse_file_entities(bold_file[0].filename,
+                                                   domains=['bids'])
```

Comment:
What does this `domains` do? I'm guessing this will cause problems with `space` entities.

Reply:
Grabbit now implements "domains" when it indexes. The default ones are 'bids' and 'derivatives', but it will add any other folders with a `layout.json` as a `domain`.

And yes I think you're correct about the `space` entity, but I was trying to workaround a grabbit bug. When fixed, I think we shouldn't have to specify a domain, and it should collapse across all.